pkg/controller/registry/resolver/operators.go

```diff
@@ -297,8 +297,6 @@ func NewOperatorFromBundle(bundle *api.Bundle, startingCSV string, sourceKey reg
 		return op, nil
 	}
 
-
```

Comment:
Looks like this file hasn't really been changed and just has noises.

Reply:
different go formatting I guess

Reply:
Would reduce noise in this PR if this was reverted back.